src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,74 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  setFilter: (value:React.ChangeEvent<HTMLSelectElement>) => void,
+  filter: string,
+  inputFilter: (event: string) => void,
+  inputFilterValue:string,
+};
+export const TodoFilter:React.FC<Props> = ({
+  setFilter,
+  filter,
+  inputFilter,
+  inputFilterValue,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            onChange={setFilter}
+            value={filter}
+            data-cy="statusSelect"
+          >
+            <option
+              value="all"
+            >
+              All
+            </option>
+            <option
+              value="active"
+            >
+              Active
+            </option>
+            <option
+              value="completed"
+            >
+              Completed
+            </option>
```

Comment:
Please cretae a separate variable for options value to list all possible options and map them